docs/packages/pkg/NLopt.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    NLopt
+
+.. index::
+  single: unsorted ; NLopt
+
+.. _pkg.NLopt:
+
+NLopt
+=====
+
+-  `Official <https://official_NLopt>`__
```

Comment:
Shoud it be https://github.com/stevengj/nlopt ?

Reply:
Yes